ansible/playbook/tasks/update.yml

```diff
@@ -18,6 +18,65 @@
       - percona-clickhouse-client
       - percona-clickhouse-server
       - pmm2-client
+      - dbaas-controller
+    pmm_rendering_dependencies:
```

Comment:
Please add a blank line before that line to give git a chance to resolve a merge conflict once we merge the next PR to the list above